src/roulette.py

```diff
@@ -126,3 +126,380 @@ def get(self, bin: int) -> Bin:
+        """
+        self.build_bins_for_straight_bets(wheel)
+        self.build_bins_for_horizontal_split_bets(wheel)
+        self.build_bins_for_vertical_split_bets(wheel)
+        self.build_bins_for_street_bets(wheel)
+        self.build_bins_for_corner_bets(wheel)
+        self.build_bins_for_line_bets(wheel)
+        self.build_bins_for_five_bet(wheel)
+        self.build_bins_for_even_money_bets(wheel)
+        self.build_bins_for_dozen_bets(wheel)
+        self.build_bins_for_column_bets(wheel)
+
+    @staticmethod
+    def build_bins_for_straight_bets(wheel: Wheel) -> None:
+        """
+        Populates **Bin** for the given  **wheel** with straight bet outcomes, each associated
+        with a unique number on the wheel.
+
+        The function iterates through numbers 0 to 36 and creates a straight bet outcome for each
+        number, each with an associated odds value of 35. Additionally, a straight bet outcome
+        for the "00" is created with the same odds.
+
+        :param wheel: The Wheel object to which the straight bet outcomes will be added.
+        :type wheel: :class:`Wheel`
+
+        **Note**: The wheel is assumed to have bins for numbers 0 to 36, and an extra bin at
+        index 37 for the "00" outcome.
+        """
+        straight_bet_odds = 35
+        straight_bet_numbers = set(range(37))
+
+        for number in straight_bet_numbers:
+            straight_bet_outcome = Outcome(str(number), straight_bet_odds)
+            wheel.addOutcome(number, straight_bet_outcome)
+        zero_zero_bet_outcome = Outcome("00", straight_bet_odds)
+        bin_index_of_zero_zero_outcome = 37
+        wheel.addOutcome(bin_index_of_zero_zero_outcome, zero_zero_bet_outcome)
+
+    @staticmethod
+    def build_bins_for_horizontal_split_bets(wheel: Wheel) -> None:
+        """
+        Populates **Bin** for  the given **wheel** with horizontal split bet outcomes,
+        each associated with a pair of adjacent numbers on the wheel.
+
+        The function iterates through pairs of adjacent numbers on the wheel and creates a
+        horizontal split bet outcome for each pair, each with an associated odds value of 17. Each
+        outcome corresponds to a bet on the line between two adjacent numbers.
+
+        :param wheel: The Wheel object to which the horizontal split bet outcomes will be added.
+        :type wheel: :class:`Wheel`
+
+
+        **Note**: The wheel is assumed to have bins for all possible numbers, and the split bet
+        outcomes are added for pairs of adjacent numbers.
+        """
+        split_bet_odds = 17
+
+        split_bet_numbers = set(range(1, 36))
+        for number in split_bet_numbers:
+            horizontal_split_bet_outcome = Outcome(
+                str(number) + "-" + str(number + 1), split_bet_odds
+            )
+            wheel.addOutcome(number, horizontal_split_bet_outcome)
+            wheel.addOutcome(number + 1, horizontal_split_bet_outcome)
```

Comment:
horizontal/left right bet should not involve third column numbers (e.g 3, 6, 9)

Reply:
Fixed.

Reply:
better update the docstring for modified code. also seems black can reformat the updated code @FarzanRashid